config/stylelint-config-carbon/rules/stylistic-issues.js

```diff
@@ -225,70 +65,298 @@ module.exports = {
-    // Require a single space or disallow whitespace after the range operator in
-    // media features.
-    'media-feature-range-operator-space-after': 'always',
-    // Require a single space or disallow whitespace before the range operator
-    // in media features.
-    'media-feature-range-operator-space-before': 'always',
-
-    // Media query list
-    // Require a newline or disallow whitespace after the commas of media query
-    // lists.
-    'media-query-list-comma-newline-after': 'always-multi-line',
-    // Require a newline or disallow whitespace before the commas of media query
-    // lists.
-    'media-query-list-comma-newline-before': 'never-multi-line',
-    // Require a single space or disallow whitespace after the commas of media
-    // query lists.
-    'media-query-list-comma-space-after': 'always-single-line',
-    // Require a single space or disallow whitespace before the commas of media
-    // query lists.
-    'media-query-list-comma-space-before': 'never',
-
     // At-rule
     // Require or disallow an empty line before at-rules (Autofixable).
     'at-rule-empty-line-before': OFF,
-    // Specify lowercase or uppercase for at-rules names (Autofixable).
-    'at-rule-name-case': 'lower',
-    // Require a newline after at-rule names.
-    'at-rule-name-newline-after': OFF,
-    // Require a single space after at-rule names.
-    'at-rule-name-space-after': 'always',
-    // Require a newline after the semicolon of at-rules.
-    'at-rule-semicolon-newline-after': 'always',
-    // Require a single space or disallow whitespace before the semicolons of at
-    // rules.
-    'at-rule-semicolon-space-before': 'never',
 
     // Comment
     // Require or disallow an empty line before comments (Autofixable).
     'comment-empty-line-before': OFF,
     // Require or disallow whitespace on the inside of comment markers.
     'comment-whitespace-inside': 'always',
 
-    // General / Sheet
-    // Specify indentation (Autofixable).
-    indentation: OFF,
-    // Limit the number of adjacent empty lines.
-    'max-empty-lines': 1,
-    // Limit the length of a line.
-    'max-line-length': OFF,
-    // Disallow end-of-line whitespace.
-    'no-eol-whitespace': true,
-    // Disallow missing end-of-source newlines (Autofixable).
-    'no-missing-end-of-source-newline': true,
+    // Shorthands
+    'declaration-block-no-redundant-longhand-properties': OFF,
   },
+  // rules: {
+  //   // Color
+  //   // Specify lowercase or uppercase for hex colors (Autofixable).
+  //   'color-hex-case': 'lower',
+  //   // Specify short or long notation for hex colors (Autofixable).
+  //   'color-hex-length': 'long',
+
+  //   // Font family
+  //   // Specify whether or not quotation marks should be used around font family
+  //   // names.
+  //   'font-family-name-quotes': 'always-where-recommended',
+
+  //   // Font weight
+  //   // Require numeric or named (where possible) font-weight values.
```

Comment:
Could you put all these commented-out ones back into their original line placement so the diff is more clear and the blame history is more intact?

Reply:
Actually I'm realizing you mentioned in the PR body

> Removed Deprecated Rule Sets
> Removed Unnecessary Disabled Rules

Are these commented out rules deprecated or unnecessary/disabled? Should we just remove them instead? That would get the diff more clean.

Also curious, how do we know that these rules are deprecated and no longer needed? Were there like a ton of console warnings or something? I just want to be sure we're still covering all these stylistic pieces.

Reply:
I removed the usage of [ deprecated-stylistic-rules ](https://stylelint.io/migration-guide/to-15#deprecated-stylistic-rules) to avoid below errors 

<img width="596" alt="image" src="https://github.com/user-attachments/assets/bc1d2ebe-c40b-40e4-a3a1-e178274bbc48">